docs/open-source-projects.md

```diff
@@ -45,41 +53,67 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 * * *
 
-## EOS Rate 
+### EOS Rate
 
-<img
-  src="https://raw.githubusercontent.com/eoscostarica/design-assets/master/logos/eosrate/eosrate--horizontal-solid-transparent-overlight.png"
-  style={{ width: "30%" }}
-/>
+> 
+<p align="left">
+	<a href="https://eosrate.io">
+		<img src="https://raw.githubusercontent.com/eoscostarica/design-assets/master/logos/eosrate/eosrate--horizontal-solid-transparent-overlight.png" width="30%"/>
+	</a>
+</p>
 
-[EOS Rate](https://eosrate.io) is an open-source dApp that allows EOS token holders to rate Block Producers (BPs) in just a few clicks. Users can access a rating system and voting portal, making it easier for token holders to make an informed decision.
+> EOS Rate is an open-source dApp that allows EOS token holders to rate Block Producers (BPs) in just a few clicks. Users can access a rating system and voting portal, making it easier for token holders to make an informed decision.
 
-- **Website** : https://eosrate.io
+- **Website** : [EOSIO Rate](https://eosrate.io)
 
-- **GitHub** : https://github.com/eoscostarica/eos-rate
+- **GitHub** : [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>]( https://github.com/eoscostarica/eos-rate)
 
-![EOS Rate](https://raw.githubusercontent.com/eoscostarica/guide.eoscostarica.io/master/static/img/OSS_screnshots/EOS_Rate.PNG)
 
-![EOS Rate](https://raw.githubusercontent.com/eoscostarica/guide.eoscostarica.io/master/static/img/OSS_screnshots/EOS_Rate_2.PNG)
+###  EOS Rate Page
+ 
+ <p align="center">
+	<a href="https://eoscostarica.io/">
+		<img src={useBaseUrl('img/OSS_screnshots/EOS_Rate.PNG')} width="100%"/>
+	</a>
+</p>
+
+<p align="center">
+	<a href="https://eoscostarica.io/">
+		<img src={useBaseUrl('img/OSS_screnshots/EOS_Rate_2.PNG')} width="100%"/>
```

Comment:
Would be better if you use a screenshot from https://eosrate.io/block-producers/costaricaeos/rate